tests/Umbraco.Tests.Common/Builders/ContentEditingBaseBuilder.cs

```diff
@@ -64,10 +66,23 @@ public override TCreateModel Build()
             _model.ParentKey = _parentKey;
         }
 
-        _model.InvariantName = _invariantName ?? Guid.NewGuid().ToString();
         _model.ContentTypeKey = _contentTypeKey;
         _model.Key = _key ?? Guid.NewGuid();
-        _model.InvariantProperties = _invariantProperties.Select(x => x.Build()).ToList();
+        _model.Properties = _invariantProperties
```

Comment:
Should this be refactored too now to align with the other changes?  I.e. so we are setting "properties" rather than "invariant properties" and "invariant name".

Reply:
I considered it - even started doing it, but then I opted to code my way around it instead, to keep the scope the PR down.

If you're OK with accepting it as-is, I will make a note of retrofitting the test code afterwards to match the new models.

Reply:
Makes sense.